src/main/webapp/index.html

```diff
@@ -10,38 +10,43 @@
 </head>
 
 <body id="content" onload="graph.initializeSlider(); graph.generateGraph()">
-  <h1>
+  <h2>
     Graph Visualization Tool
-  </h1>
-  <div id="search-box">
-    <label for="layers">Radius from filtered node(s) (0-20, default is 3):</label>
-    <input type="number" id="num-layers" name="layers" min="0" max="20" value="3">
-    <label for="search-name">Name of node(s) to filter for (If you enter multiple, please separate with commas!) </label>
-    <input type="text" id="node-name-filter" name="search-name" placeholder="nodeName1,nodeName2"></input>
-    <label for="token-name">Name of token to filter for</label>
-    <input type="text" id="token-name-filter" name="token-name" placeholder="Token Name"></input>
-    <button onclick="graph.generateGraph()">Get Graph</button>
-    <div>
-      <label>Zoom on Node in Graph: </label>
-      <input type="text" id="node-search" placeholder="Node Name"></input>
-      <button id="search-button">Find Node</button>
-      <label id="node-error" class="error-text"></label>
+  </h2>
+  <div id="input-boxes">
+    <!-- Box for filtering the graph -->
+    <div id="search-box" class="input-child">
+      <h4 class="search-title">Filter the Graph</h4>
+      <label for="layers">Radius from filtered node(s) (0-20, default is 3):</label>
```

Comment:
Do you think we should rethink this 20 thing now?

Reply:
what do you think it should be?

Reply:
Maybe the actual max depth? But then we would have to add a backend parameter.